### PR TITLE
Tweak: Added a vertical alignment control to icons in Icon List widget [ED-9265]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by TzviRabinovitch — DISMISSED** [review]

File: assets/dev/scss/frontend/widgets/icon-list.scss [hunk 1]
```diff
@@ -55,9 +55,11 @@
 		}
 
 		&, a {
-			display: flex;
-			align-items: center;
 			font-size: inherit;
+			display: flex;
+
+			$e-icon-vertical-align-default: center;
+			align-items: var(--icon-vertical-align, $e-icon-vertical-align-default);
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply assets/dev/scss/frontend/widgets/icon-list.scss:62]: Done

File: includes/widgets/icon-list.php [hunk 2]
```diff
@@ -487,6 +487,59 @@ protected function register_controls() {
...
+						'icon' => 'eicon-v-align-top',
+					],
+					'center' => [
+						'title' => esc_html__( 'Center', 'elementor' ),
+						'icon' => 'eicon-v-align-middle',
+					],
+					'flex-end' => [
+						'title' => esc_html__( 'End', 'elementor' ),
+						'icon' => 'eicon-v-align-bottom',
+					],
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply includes/widgets/icon-list.php:507]: @avivu Can you please guide us? 
It isn't 100% clear inside the specs. 

I have used: **Top, Center, Bottom.**

File: includes/widgets/icon-list.php [hunk 3]
```diff
@@ -487,6 +487,59 @@ protected function register_controls() {
...
+						'max' => 15,
+					],
+					'em' => [
+						'min' => -1,
+						'max' => 1,
+						'step' => 0.1,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}}' => '--icon-vertical-align-offset: {{SIZE}}{{UNIT}};',
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply includes/widgets/icon-list.php:538]: Changed

File: tests/playwright/sanity/includes/widgets/icon-list.test.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
+
+test.describe( 'Icon List', () => {
+	test( 'Test vertical alignment of the icons', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost();
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply tests/playwright/sanity/includes/widgets/icon-list.test.js:8]: Updated.

File: includes/widgets/icon-list.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (hein-obox) [reply includes/widgets/icon-list.php:507]: @StuartElementor Aviv indicated that we can use Start, Center, End.

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**